src/components/grid/grid.stories.mdx

```diff
@@ -240,6 +226,97 @@ This example is best viewed in the Canvas tab using full-screen mode with device
   </Story>
 </Preview>
 
+## Custom spacings
+This example is best viewed in the Canvas tab using full-screen mode with device or viewport emulation.
+<Preview>
+  <Story name="custom spacings" parameters={{ info: { disable: true }}} >
+    <GridContainer m='20px' gridGap='5px'>
+      <GridItem
+        p='20px 5px 20px 0'
```

Comment:
I didn't know you could use the `styled-sytem` `p` prop like that - thanks for that 👍